mindsdb/integrations/handlers/openai_handler/openai_handler.py

```diff
@@ -201,11 +204,8 @@ def predict(self, df: pd.DataFrame, args: Optional[Dict] = None) -> pd.DataFrame
 
         pred_args = args['predict_params'] if args else {}
         args = self.model_storage.json_get('args')
-        args['api_base'] = pred_args.get(
-                    'api_base',
-                    args.get(
-                        'api_base', os.environ.get('OPENAI_API_BASE', OPENAI_API_BASE)
-                    ))
+        connection_args = self.engine_storage.get_connection_args()
+        args['api_base'] = connection_args.get('api_base') or args.get('api_base') or os.environ.get('OPENAI_API_BASE', OPENAI_API_BASE)
```

Comment:
I think this is missing `or pred_args.get('api_base')`

Reply:
Yes, adding it